Resources/public/js/recommendationsupport.js

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+(function (global, doc) {
+    var eZ = global.eZ = global.eZ || {};
+
+    /**
+     * Contains logic needed to fetch and display YooChoose recommendations
+     *
+     * @class
+     */
+    eZ.RecommendationSupport = function () {};
+
+    /**
+     * Default settings
+     *
+     * @type {Array}
+     */
+    eZ.RecommendationSupport.prototype.config = {
```

Comment:
while this works, it's a bit unsual to put the configuration in the prototype because the very same config object will be shared amongst the different instances so changing the configuration in one of them, actually changes it for all of them. It's usually better to initialize the configuration in the constructor.